src/TestInfrastructure/Helpers/MefTestHelpers.cs

```diff
@@ -33,9 +33,9 @@ namespace SonarLint.VisualStudio.Integration.UnitTests
     public static class MefTestHelpers
     {
         [DebuggerStepThrough]
-        public static Export CreateExport<T>() where T: class
+        public static Export CreateExport<T>(T value = null) where T : class
```

Comment:
Why was this needed? there is a method below that accepts a value

Reply:
Hmm, true, I just got thrown off by the fact that that method received an object. Will revert this change